README.md

```diff
@@ -10,9 +8,19 @@ Shunter is a [Node.js][node] application built to read JSON and translate it int
 [![Dependencies][shield-dependencies]][info-dependencies]
 [![LGPL-3.0 licensed][shield-license]][info-license]
 
-![Shunter as a proxy](docs/diagram.png)
+Shunter is a [Node.js][node] application built to read JSON and translate it into HTML.
+
+It helps you create a decoupled front end which can serve traffic from one or more backend applications - great for use in multi-language, multi-disciplinary teams or just to make your project more flexible and future-proofed.
+
+# Key Features
+- Enforces decoupling of templates from underlying applications
+- Enables multiple applications to use the same unified front end
+- Makes full site redesigns or swapping out back end applications a doddle
+- Completely technology-agnostic; if your application outputs JSON, it can work with Shunter
+- Asset concatenation, minification, cache-busting, and other performance optimisations built-in
+- Outputs any type of content you like, e.g. HTML, RSS, RDF
+- Well-tested and supported, serving high-traffic sites across nature.com
```

Comment:
Do we want to link to our gross current home page? I think it gives the wrong impression. `nature.com` will expand to a link I think